src/vuex/modules/session.js

```diff
@@ -15,7 +15,7 @@ export default () => {
   const state = {
     developmentMode: config.development, // can't be set in browser
     experimentalMode: config.development, // development mode, can be set from browser
-    insecureMode: false, // show the local signer
+    insecureMode: true, // show the local signer
```

Comment:
don't forget to remove this again ^^

Reply:
dangerous to set it in code. better use the query parameter: `?insecure=true`

Reply:
> don't forget to remove this again ^^

Yeah, Colin was farsighted and wrote a test to check that it's false :-)

Reply:
Good guy Colin